src/main/steps/applicant1/hub-page/joint/form.njk

```diff
@@ -23,14 +23,13 @@
       name: applicantApplyForConditionalOrderStarted,
       value: "Yes"
     }) }}
-  {% elif displayState.state() == 'AwaitingFinalOrder' %}
-      {{ govukButton({
-        text: button.applyForFinalOrder,
-        classes: form.submit.classes,
-        type: "submit",
-        name: "TODO",
-        value: "Yes",
-        preventDoubleClick: true
-      }) }}
+    {% elif displayState.state() == 'AwaitingFinalOrder' or 'AwaitingJointFinalOrder' %}
+    {{ govukButton({
+      text: button.applyForFinalOrder,
+      classes: form.submit.classes,
+      href: awaitingFinalOrder.buttonLink,
+      type: "submit",
+      preventDoubleClick: true
+    }) }}
```

Comment:
```suggestion
    {% elif displayState.state() == 'AwaitingFinalOrder' or 'AwaitingJointFinalOrder' %}
    {{ govukButton({
        text: button.applyForFinalOrder,
        classes: form.submit.classes,
        href: awaitingFinalOrder.buttonLink,
        type: "submit",
        preventDoubleClick: true
    }) }}
```
what does this button do? If it changes the state, then we can use the homeGetController to determine where the user gets sent to after clicking it, rather than using href

Reply:
Hi @lara-mpt this button takes the user to the the finalising your application page, it does not change the state I believe. The finalising your application page has a post controller which then changes the state to either APPLICANT2_FINAL_ORDER_REQUESTED or FINAL_ORDER_REQUESTED.